movies_app.py

```diff
@@ -0,0 +1,8 @@
+class Movies:
+	_statement = f"Rental Record For {self.customer} \n You Owe 0.0 \n You earned 0"
```

Comment:
I would move this statement out of the the class and back into the statement function.
Then, each time there is a new field you have to be able to adjust, add a new variable to the init and do the string concatenation on self to fill it in.